lib/FullScreenView/FullScreenView.js

```diff
@@ -0,0 +1,78 @@
+import React, {
+  useRef,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Pane,
+  PaneMenu,
+  IconButton,
+  Button,
+} from '@folio/stripes/components';
+
+import css from './FullScreenView.css';
+
+export const FullScreenView = ({
+  id = 'full-screen-view',
+  children,
+  paneTitle,
+  onCancel,
+}) => {
+  const headerCloseButtonRef = useRef();
+
+  useEffect(() => headerCloseButtonRef.current.focus(), []);
+
+  const firstMenu = (
+    <PaneMenu>
+      <FormattedMessage id="stripes-components.cancel">
+        {ariaLabel => (
+          <IconButton
+            data-test-header-close-button
+            ariaLabel={ariaLabel}
+            icon="times"
+            ref={headerCloseButtonRef}
+            onClick={onCancel}
+          />
+        )}
+      </FormattedMessage>
+    </PaneMenu>
+  );
+
+  return (
+    <div
+      id={id}
+      data-test-full-screen-view
+    >
+      <Pane
+        defaultWidth="100%"
+        paneTitle={paneTitle}
+        firstMenu={firstMenu}
+        actionMenu={({ onToggle }) => (
+          <Button
+            buttonStyle="dropdownItem fullWidth"
+            onClick={onToggle}
+          />
+        )}
+      >
+        <div
+          data-test-full-screen-view-content
+          className={css.viewContent}
+        >
+          {children}
+        </div>
+      </Pane>
+    </div>
+  );
+};
+
+FullScreenView.propTypes = {
+  id: PropTypes.string,
+  onCancel: PropTypes.func.isRequired,
+  paneTitle: PropTypes.node.isRequired,
+  children: PropTypes.oneOfType([
```

Comment:
Can it be replaced with just `children: PropTypes.node`?

Reply:
@VictorSoroka1 This approach is used in FullScreenForm. Please, could you clarify the difference between them in case of this property?

Reply:
Definitely yes, thanks for noticing

Reply:
Great, thank you for pointing that we have it in another place. I will change there too.

Reply:
@VictorSoroka1, maybe, I should fix there in a scope of my story?

Reply:
It would be great.